exporter/zipkinexporter/zipkin.go

```diff
@@ -103,7 +103,7 @@ func ZipkinExportersFromViper(v *viper.Viper) (tps []consumer.TraceConsumer, mps
 		return nil, nil, nil, fmt.Errorf("cannot configure Zipkin exporter: %v", err)
 	}
 	tps = append(tps, zle)
-	doneFns = append(doneFns, zle.stop)
+	doneFns = append(doneFns, zle.Stop)
```

Comment:
It should be safe to delete entire `ZipkinExportersFromViper`. It should not be used anywhere.

Reply:
I'll add a todo there are lots of little things that need to be cleaned up. Ex: the name being stored for most exporters isn't the full name but just the type of the exporter.

Reply:
https://github.com/open-telemetry/opentelemetry-service/issues/263